migrations/versions/26a83ff1e6ad_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
+
+Revision ID: 26a83ff1e6ad
+Revises: aa4b21539f5b
+Create Date: 2016-06-16 17:34:53.524263
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '26a83ff1e6ad'
+down_revision = 'aa4b21539f5b'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('language')
+    op.drop_constraint(u'session_language_id_fkey', 'session', type_='foreignkey')
```

Comment:
op.drop_constraint .... move this line above op.drop_table to avoid db upgrade issues.